app/views/admin/components/MaintenanceModal.vue

```diff
@@ -0,0 +1,55 @@
+<script>
+  export default Vue.extend({
+    data() {
+     return {
+       hour: 1,
+     }
+    },
+    methods: {
+      submit: async function(){
+        await fetch('/admin/maintenance-time', {
+          method: 'POST',
+          body: JSON.stringify({hour: this.hour}),
+          headers: {'Content-Type': 'application/json'}
+        })
+      }
+    }
+  })
+</script>
+<template lang="pug">
+#modal-base-flat
+  .body
+    .alert
+      p MAINTENANCE MODE MAKES WEBSITE UNVISITABLE!
```

Comment:
any native speakers pls to update this stupid `unvisitable` ,

Reply:
Inaccessible